core/src/test/java/org/frankframework/management/bus/BusTestEndpoints.java

```diff
@@ -46,6 +46,8 @@ public Message<String> throwException(Message<?> message) {
 			new ConfigurationException("cannot configure",
 				new IllegalStateException("something is wrong")));
 		switch (type) {
+			case NOT_FOUND:
+				throw new ResourceNotFoundException("Resource not found");
```

Comment:
Looks like the formatting is different then the other lines around it.

Reply:
Yeah indeed, my IntelliJ seems to do this automatically so I will look into this